war/src/main/webapp/css/google-fonts/roboto/css/roboto.css

```diff
@@ -0,0 +1,69 @@
+/* roboto-300 - greek_latin-ext_latin_vietnamese_cyrillic_greek-ext_cyrillic-ext */
```

Comment:
To replace `@import url(https://fonts.googleapis.com/css?family=Roboto:400,300,500,900,700);`